src/main/java/org/openstreetmap/atlas/checks/validation/intersections/BuildingRoadIntersectionCheck.java

```diff
@@ -128,6 +131,7 @@ public boolean validCheckForObject(final AtlasObject object)
                 ignoreTags());
         final CheckFlag flag = new CheckFlag(getTaskIdentifier(building));
         flag.addObject(building);
+        flag.addFixSuggestion(FeatureChange.remove(CompleteEntity.shallowFrom(building)));
```

Comment:
@ladwlo Like we discussed offline, this may not be the best way to resolve buildings intersecting roads. Please see my comment on https://github.com/osmlab/atlas-checks/issues/392. I feel like this suggestion would not be acceptable in most cases. We may need a further investigation on this.